sdk/test-utils/recorder-new/test/testProxyTests.spec.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env } from "@azure-tools/test-recorder";
+import { createPipelineRequest } from "@azure/core-rest-pipeline";
+import { ServiceClient } from "@azure/core-client";
+import { recorderHttpPolicy, TestProxyHttpClient } from "../src";
+import { expect } from "chai";
+
+const setTestMode = (
+  mode: "record" | "playback" | undefined
+): "record" | "playback" | undefined => {
+  env.TEST_MODE = mode;
+  console.log(`TEST_MODE = ${mode}`);
+  return mode as "record" | "playback" | undefined;
+};
+
+const TEST_SERVER_URL = `http://127.0.0.1:8080`;
+
+// These tests require the following to be running in parallel
```

Comment:
Is it possible to start the server from code (e.g., Mocha `before()` hook)?

Reply:
probably not for browser tests...

Reply:
> probably not for browser tests...

Yes, that's the reason for putting this separately.